lib/src/widgets/chat_bubble_widget.dart

```diff
@@ -29,9 +29,9 @@ import 'profile_circle.dart';
 import 'reply_message_widget.dart';
 import 'swipe_to_reply.dart';
 
-class ChatBubbleWidget extends StatelessWidget {
+class ChatBubbleWidget extends StatefulWidget {
```

Comment:
What's the reason to change this class to statefulWidget ?

Reply:
`GlobalKey` requires a widget to be stateful.

Reply:
can you please share more details?

Reply:
@vatsaltanna We need access to this widget's state using the global key. But globalkey requires a widget to be stateful when we want to access it's state because, createState is only called for stateful widget.